Cue/app/tabs/library/LibraryHome.js

```diff
@@ -76,7 +80,22 @@ class LibraryHome extends React.Component {
   componentWillReceiveProps(newProps: Props) {
     this.setState({
       refreshing: false
-    })
+    })
+
+    if (newProps.inaccessibleDecks && newProps.inaccessibleDecks.length > 0) {
+      newProps.inaccessibleDecks.forEach(deck => {
+        Alert.alert(
+          ('The deck "' + deck.name + '" is no longer available'),

```

Comment:
Use curly quotes: `“ ... ”`, Title Case on iOS, and maybe clarify that the deck is no longer available *from the original owner*?